resources/language/en_us.ini

```diff
@@ -0,0 +1,1565 @@
+clear.loadMoneyError = "§cThere was an error while retrieving your money."
+; this message gets displayed when the command executor has not enough money to clear a plot
+; {%0} = the currency of the economy provider
+; {%1} = the price how much it costs to clear a plot
+; {%2} = the money the command executor is short on
+clear.notEnoughMoney = "§cYou need §6{%2}{%0} §cto clear this plot (costs §6{%1}{%0}§c)."
+; this message gets displayed when the command executor's money was successfully reduced
+; {%0} = the currency of the economy provider
+; {%1} = the price how much it costs to clear a plot
+clear.chargedMoney = "§eYou have been charged §6{%1}{%0} §eto clear the plot."
+; this message gets displayed before the plot is cleared
+clear.start = "§7The plot is now being cleared..."
+; this message gets displayed after the plots are successfully merged together
+; {%0} = time in which the plot was cleared
+clear.finish = "§7The plot was successfully cleared in §6{%0} §7"
+
+
+; define the name of the “/plot denied“ subcommand
+denied.name = "denied"
+; define aliases of the “/plot denied“ subcommand
+denied.alias = "[]"
+; define the description of the “/plot denied“ subcommand
+denied.description = "See the denied players of your plot."
+; define the usage message of the “/plot denied“ subcommand
+denied.usage = "§c/p denied"
+; this message gets displayed when the command executor hasn't the permissions to use the “/plot denied“ subcommand
+denied.permissionMessage = "§cYou don't have the permission to use this command."
+; this message gets displayed when the command executor isn't online (for example, if the command executor is the console)
+denied.senderNotOnline = "§cThis command must be run in-game."
+; this message gets displayed when the command executor isn't in a plot world
+denied.noPlotWorld = "§cYou are not in a plot world."
+; this message gets displayed when the command executor is not on a plot
+denied.noPlot = "§cYou are not on a plot."
+; this message gets displayed when there are no denied players on the plot
+denied.noDeniedPlayers = "§cThere are no denied players on this plot."
+; this message contains the list of denied players on the plot
+; {%0} = the list of denied players
+denied.success = "§7The following players are denied on this plot:\n§r§6{%0}"
+; this message is the format of the list of the denied players
+; {%0} = the name of the denied player
+; {%1} = the formatted add time as denied player (@see denied.success.list.addTime.format)
+denied.success.list = "§6{%0} §7since {%1}"
+; this message is the format message of the add time as denied player
+; {%0} = the day of the add as denied player
+; {%1} = the month of the add as denied player
+; {%2} = the year of the add as denied player
+; {%3} = the year of the add as denied player
+; {%4} = the minute of the add as denied player
+; {%5} = the second of the add as denied player
+denied.success.list.addTime.format = "{%0}.{%1}.{%2} um {%3}:{%4}:{%5} Uhr"
```

Comment:
This wasn't translated. I think it should be `{%0}.{%1}.{%2} {%3}:{%4}:{%5} o'clock`, so it would be `PlayerX since 17.04.2022 07:31:13 o'clock`? (But you probably know better)